### PR TITLE
feat(dapps): display transaction fees and supported networks

### PR DESCRIPTION
**Pull request by bakoushin**
### Description

* Display fees in WC bottom sheets (Celo and Ethereum):
  * Sign transaction
  * Send transaction
* Display network name in the body text
* Display matched supported networks chips in the WC connect bottom sheet
* Update copy/button labels/overall appearance according to designs

| Connect wallet | Fee currency: Celo | Fee currency: cUSD |
|--------|--------|--------|
|   |  Simulator Screenshot - iPhone SE (3rd generation) - 2023-12-14 at 19 03 50 | Simulator Screenshot - iPhone SE (3rd generation) - 2023-12-14 at 19 12 28 | 

### Test plan

* Tested manually
  * Connect wallet: Celo Mainnet, Alfajores, Ethereum ([Uniswap dapp](http://app.uniswap.org/))
  * Send transaction: Celo Mainnet, Alfajores ([Mento dapp](https://app.mento.org))
* Updated unit tests

### Related issues

- Fixes RET-944

### Backwards compatibility

Y

### CODE REVIEW

File: locales/base/translation.json [hunk 1]
```diff
@@ -1073,12 +1073,21 @@
     "signPayload": "{{dappName}} would like to verify ownership of your wallet.",
     "signTransactionTitle": "Sign transaction",
     "signTransaction": "{{dappName}} would like to sign a Celo transaction.",
+    "signDappTransaction": "{{dappName}} would like to sign a transaction on the {{networkName}} network.",
+    "signDappTransactionUnknownNetwork": "{{dappName}} would like to sign a transaction on the unknown network.",
```
Comment (kathaypacific) [inline locales/base/translation.json:1077]:
```suggestion
    "signDappTransactionUnsupportedNetwork": "{{dappName}} would like to sign a transaction on an unsupported network.",
```

File: locales/base/translation.json [hunk 2]
```diff
@@ -1073,12 +1073,21 @@
...
     "sendTransactionTitle": "Send transaction",
     "sendTransaction": "{{dappName}} would like to send a Celo transaction.",
+    "sendDappTransaction": "{{dappName}} would like to send a transaction on the {{networkName}} network.",
+    "sendDappTransactionUnknownNetwork": "{{dappName}} would like to send a transaction on the unknown network.",
```
Comment (kathaypacific) [inline locales/base/translation.json:1081]:
```suggestion
    "sendDappTransactionUnsupportedNetwork": "{{dappName}} would like to send a transaction on an unsupported network.",
```

File: src/walletConnect/screens/ActionRequest.tsx [hunk 3]
```diff
@@ -133,6 +140,9 @@ function ActionRequest({
         request={pendingAction}
         preparedTransaction={preparedTransaction}
       />
+      {preparedTransaction && (
```
Comment (kathaypacific) [inline src/walletConnect/screens/ActionRequest.tsx:143]:
```suggestion
      {useViem && preparedTransaction && (
```

File: src/walletConnect/screens/EstimatedNetworkFee.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      ? getTokenId(networkId, transaction.feeCurrency)
+      : getTokenId(networkId)
+
+  const feeTokenInfo = useTokenInfo(feeTokenId)
+
+  const networkName = NETWORK_NAMES[networkId]
+
+  if (!transaction.gas || !transaction.maxFeePerGas || !feeTokenInfo || !networkName) {
+    // insufficient information to display fee details
+    return null
```
Comment (kathaypacific) [inline src/walletConnect/screens/EstimatedNetworkFee.tsx:34]: this is unexpected right? maybe we can log a warning or error?

**Review by kathaypacific — APPROVED** [review]
LGTM!!  ✨

File: src/walletConnect/screens/EstimatedNetworkFee.tsx (hunk 4 above)
Comment (bakoushin) [reply src/walletConnect/screens/EstimatedNetworkFee.tsx:34]: yes, normally this shouldn't happen. added a warning log.

**Pull request merged**